spec/factories/role.rb

```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
   factory :role do |role|
-    role.name             'super-danger-admin'
+    name { Faker::Internet.slug(FFaker::Job.title) }
```

Comment:
So we added `ffaker` gem for just the job title? :grimacing: 

Wouldn't it be easier to just stick with the `faker` gem and add a custom module like I did for Contiki:

``` ruby
# lib/faker/contiki.rb
require 'faker'

module Faker
  class Contiki < Base
    flexible :Contiki

    class << self
      def season
        fetch('contiki.season')
      end
    end
  end
end
```

which is loaded via an initializer:

``` ruby
# config/initializers/faker.rb
if defined?(Faker)
  require 'faker/contiki'
end
```

and pulls it's data from a locale file:

``` yaml
# config/locales/en.faker.yml
en:
  faker:
    contiki:
      season:
        - summer
        - winter
        - year_round
```

So my recommendation would be to add a `Faker::DSDS` module - or go with the generic `Faker::Job` if you prefer.